single-signon.yaml

```diff
@@ -0,0 +1,21 @@
+# This file is copied from https://github.com/CCI-MOC/rhosp-director-config/blob/master/templates/single-signon.yaml
```

Comment:
Note that this file was last updated four years ago, so may no longer represent the correct tripleo configuration. When in doubt, check the docs and current templates.